src/main/java/pl/betoncraft/betonquest/BetonQuest.java

```diff
@@ -414,6 +426,8 @@ public void onEnable() {
 			public void run() {
 				// Load all events and conditions
 				loadData();
+				// Load global tags and points
+				globalData = new GlobalData();
```

Comment:
You're creating it on plugin start, but then you're not reloading it in the `reload` method. It's no big deal, but the plugin won't pull the newest changes made manually in the database on reload. For some users this might be inconvenient.

Also if you're going to implement this, remember to clear old data or it will get doubled.

Reply:
As far as I see player data is also only loaded on startup and not pulled agian on reload 
(or did I miss something??? then please tell me... 🤔).
So for now I'll leave it as it isto at least keep it consistent.

Maybe at some point I'll have time to implement it for both player data and global data.

Reply:
No, actually you're right. I forgot I removed player data reload because of performance reasons on bigger servers. So it's perfect right now.